src/utils/errors.ts

```diff
@@ -66,9 +68,21 @@ export class MatrixRTCFocusMissingError extends ElementCallError {
 export class ConnectionLostError extends ElementCallError {
   public constructor() {
     super(
-      "Connection lost",
+      "ConnectionLostError",
```

Comment:
Just noticing this: Is it possible for ElementCallError to determine its own name via [`obj.constructor.name`](https://stackoverflow.com/a/8715426), rather than requiring each subclass to specify it manually?